### PR TITLE
[Feature][kafka] Add arg  poll.timeout  for interval poll messages

### PR DESCRIPTION
**Pull request by CosmosNi**
Add arg  poll.timeout  for interval poll messages

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by CosmosNi** [comment]
@iture123 @hailin0 @Hisoka-X PTAL,thanks.

File: docs/en/connector-v2/source/kafka.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ They can be downloaded via install-plugin.sh or from the Maven central repositor
...
+| poll.timeout                        | Long                                                                        | No       | 10000L                   | The interval for poll messages.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (Hisoka-X) [inline docs/en/connector-v2/source/kafka.md]: unit?

File: docs/en/connector-v2/source/kafka.md [hunk 2]
```diff
@@ -32,26 +32,27 @@ They can be downloaded via install-plugin.sh or from the Maven central repositor
...
+| poll.timeout                        | Long                                                                      | No       | 10000L[millis]                 | The interval for poll messages.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (Hisoka-X) [inline docs/en/connector-v2/source/kafka.md]:
```suggestion
| poll.timeout                        | Long                                                                      | No       | 10000               | The interval(millis) for poll messages.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

File: docs/zh/connector-v2/source/Kafka.md [hunk 3]
```diff
@@ -40,6 +40,7 @@
...
 | commit_on_checkpoint                | Boolean                             | 否    | true                     | 如果为 true，消费者的偏移量将会定期在后台提交。                                                                                                                                                                                                                                                                                              |
+| poll.timeout                        | Long                                 | 否    | 10000L[millis]                    | kafka主动拉取时间间隔。                                                                                                                                                                                                                                                                                                          |
```
Comment (Hisoka-X) [inline docs/zh/connector-v2/source/Kafka.md]:
```suggestion
| poll.timeout                        | Long                                 | 否    | 10000                    | kafka主动拉取时间间隔(毫秒)。                                                                                                                                                                                                                                                                                                          |
```

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @CosmosNi

**Review by hailin0 — APPROVED** [review]
LGTM

**Pull request merged**